org.opentosca.container.api/src/org/opentosca/container/api/controller/SituationsController.java

```diff
@@ -103,6 +106,20 @@ public Response createSituation(final SituationDTO situation) {
 
 		return Response.ok(instanceURI).build();
 	}
+	
+	@DELETE
+    @Consumes({ MediaType.TEXT_PLAIN })
+    @Path("/situations/{situation}")
+	public Response deleteSituation(@PathParam("situation") final Long situationId) {
+	    
+	    boolean removed = this.instanceService.removeSituation(situationId);
+	    
+	    if(removed){	        
+	        return Response.ok().build();
+	    } else {
+	        return Response.status(403).build();
```

Comment:
A "Forbidden" can raise some eyebrows here. Consider explaining why that is appropriate in a comment or use a status code like 412_PRECONDITION_FAILED